gke/Jenkinsfile

```diff
@@ -66,7 +68,9 @@ pipeline {
 	    steps {
 	        container(name: 'kaniko', shell: '/busybox/sh') {
 		    sh '''#!/busybox/sh
-		    /kaniko/executor -f `pwd`/Dockerfile -c `pwd` --context="gs://${JENKINS_TEST_BUCKET}/${BUILD_CONTEXT}" --destination="${GCR_IMAGE}" --build-arg JAR_FILE="${APP_JAR}"
+                    pwd
```

Comment:
Are these two new lines intended or were they added for debugging?

Reply:
Yes. Good catch!